app/services/measurements_creator.rb

```diff
@@ -2,6 +2,13 @@ class MeasurementsCreator
   SLICE_SIZE = 500
 
   def self.call(stream, measurements_attributes)
```

Comment:
In the spirit of unlegacing the app we should probably cover the entire method in tests